pages/content/amp-dev/documentation/guides-and-tutorials/start/create/index.md

```diff
@@ -1,14 +1,18 @@
 ---
 $title: Create your first AMP page
 $order: 0
-description: 'Not sure how to get started? In this tutorial, you’ll learn how to create a basic AMP HTML page, how to stage it and validate that it’s AMP compliant, and finally ...'
+description: 'In this tutorial, we’ll introduce you to the basics of an AMP page!'
 formats:
   - websites
 numbered: 0
 $hidden: true
-author: pbakaus
-contributors:
-  - bpaduch
+author: crystalonscript
 ---
 
-Not sure how to get started? In this tutorial, you’ll learn how to create a basic AMP HTML page, how to stage it and validate that it’s AMP compliant, and finally how to get it ready for publication and distribution.
+In this tutorial, we’ll introduce you to the basics of an AMP page. Follow these steps to get started learning how AMP is unique, checking if a page is valid, and taking next steps to publishing it to the web.
+
+AMP pages that pass validation are more likely to meet [Page Experience metrics](https://amp.dev/about/page-experience/) and enable caching! If you are uninterested in AMP cache benefits or unable to switch to fully valid AMP but still want to use AMP's components, check out [Bento AMP](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/?format=websites). Bento allows the use of AMP components everywhere on the web! It is also okay to create invalid AMP pages, but it isn't a directly supported use case and there's no guarantee that invalid AMP page will work correctly.
```

Comment:
Suggestion: Link `AMP cache benefits` to a relevant page in case it's not obvious or readily accessible what these are?